### DISCUSSION TITLE
icone

### DISCUSSION THREAD

**Discussion by Nik71git** [maxill1/ialarm-mqtt#44] [Ideas]
è possibile inserire nello yaml di configurazione (parlo di addon per Home Assistant) le icone material design oppure è meglio farlo nelle personalizzazioni di HA? mi piacerebbe avere le icone variant per le window open/close

**Comment by maxill1** [comment]
intendi per le singole zone?

Mi pare di ricordare che si possono dare nello yaml della versione standard (devo testarlo), ma l'icona è una e dovrei capire come gestire la coppia aperta chiusa (template?). 

Sull'addon devo aggiungere sicuramente 'icon" allo schema di 'zones' (https://github.com/maxill1/addons/blob/master/ialarm-mqtt/config.yaml). Appena riesco ci do un occhio.
